stable/postgresql/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: postgresql
-version: 5.1.1
+version: 5.1.2
```

```suggestion
version: 5.1.3
```

The version 5.1.2 is already merged in master. Can you bump the version again?